distribution/docs/src/main/resources/_contents/_endpoints/available-endpoints-contents.adoc

```diff
@@ -0,0 +1,15 @@
+
+=== Available Endpoints
+
+The following endpoints are available in a standard installation of ${branding}:
+
+<<_application_upload_endpoint,Application Upload Endpoint>>:: Uploads new/upgraded applications to the system.
+<<_catalog_rest_endpoint,Catalog REST Endpoint>>:: Allows clients to perform CRUD operations on the Catalog using REST, a simple architectural style that performs communication using HTTP. _implements https://www.w3.org/2001/sw/wiki/REST[REST specification].
```

Comment:
Missing `_` after _implements https://www.w3.org/2001/sw/wiki/REST[REST specification]_.

Same problem for IdP Endpoint.

Also shouldn't all those _implements_ be capitalized?